providers/fips/fipsprov.c

```diff
@@ -351,3 +408,74 @@ const OSSL_PROVIDER *FIPS_get_provider(OPENSSL_CTX *ctx)
 
     return fgbl->prov;
 }
+
+void *CRYPTO_malloc(size_t num, const char *file, int line)
+{
+    return c_CRYPTO_malloc(num, file, line);
```

Comment:
What happens if these c_CRYPTO_ calls dont get resolved and they are NULL?

Reply:
yeah, i'm tracing down why ```c_CRYPTO_zalloc``` is NULL.  to answer your question, you crash:
```
Program received signal SIGSEGV, Segmentation fault.
0x0000000000000000 in ?? ()
(gdb) bt 3
#0  0x0000000000000000 in ?? ()
#1  0x00007ffff6bef492 in CRYPTO_zalloc (num=248, 
    file=0x7ffff6bf3789 "crypto/context.c", line=118)
    at providers/fips/fipsprov.c:419
#2  0x00007ffff6bb80b0 in OPENSSL_CTX_new () at crypto/context.c:118
(More stack frames follow...)
(gdb) p c_CRYPTO_zalloc 
$1 = (OSSL_CRYPTO_zalloc_fn *) 0x0
```